package.json

```diff
@@ -10,22 +10,25 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@dashx/react-native": "file:../dashx-react-native",
```

Comment:
@Charan017 please remove this local repo reference. Please point to an appropriate version of `@dashx/react-native` repo

Reply:
Done